fem/bilinearform_ext.cpp

```diff
@@ -874,6 +878,9 @@ void FABilinearFormExtension::Assemble()
             static_cast<const L2ElementRestriction*>(elem_restrict);
          const L2FaceRestriction *restF =
             static_cast<const L2FaceRestriction*>(int_face_restrict_lex);
+         MFEM_VERIFY(
+            !dynamic_cast<const NCL2FaceRestriction*>(int_face_restrict_lex),
+            "Full Assembly not yet supported on NCMesh.");
```

Comment:
Ditto

Reply:
done in [a84b74d](https://github.com/mfem/mfem/pull/2840/commits/a84b74d625e17ac6ecc336921d982daf06bf2e9a)